indico/htdocs/sass/modules/_registrationform.scss

```diff
@@ -1304,3 +1304,70 @@ textarea.regFormDescriptionInputMgmt {
         }
     }
 }
+
+
+.invitation-list {
+    .invitation-table {
```

Comment:
Similar but not the same. But I guess we could refactor it at some point and extract the common parts.